Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -17,13 +17,26 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_FIELD_VALUE_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_FIELD_VALUE_H_
 
+#include <stdint.h>
+
+#include <map>
 #include <memory>
+#include <string>
 #include <vector>
 
+#include "Firestore/core/include/firebase/firestore/blob.h"
+#include "Firestore/core/include/firebase/firestore/geo_point.h"
+#include "Firestore/core/src/firebase/firestore/model/timestamp.h"
+
 namespace firebase {
 namespace firestore {
 namespace model {
 
+struct ServerTimestamp {
+  Timestamp local;
```

Comment:
Please preserve the more descriptive names we had before. `local_write_time` conveys more than just `local`.

Reply:
done